Filters/exceptions.txt

```diff
@@ -1,3 +1,6 @@
+! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/1303
+@@||capi.connatix.com^|
+@@||cd.connatix.com^|
```

Comment:
`||cd.connatix.com^` / `||capi.connatix.com^` are in DNS filter. Fix in `exclusions.txt`
This is not pure ad service, so beter do not block by DNS if this causes problems.